airflow/config_templates/config.yml

```
@@ -492,6 +492,14 @@ core:
       type: string
       example: ~
       default: "Disabled"
+    max_templated_field_size:
+      description: |
+        The maximum size of the rendered template field in bytes. If the value to be stored in the
```

Nit: not bytes, characters.

```suggestion
        The maximum length of the rendered template field. If the value to be stored in the
```